elements/bulbs-video/components/revealed.test.js

```diff
@@ -613,13 +620,14 @@ describe('<bulbs-video> <Revealed>', () => {
         let vastUrl = Revealed.prototype.vastUrl.call({
           cacheBuster: cacheBusterStub,
           vastTest: vastTestStub,
+          utmTest: utmTestStub,
           props: {},
         }, videoMeta);
         let parsed = url.parse(vastUrl, true);
         expect(parsed.protocol).to.eql('https:');
         expect(parsed.host).to.eql('pubads.g.doubleclick.net');
         expect(parsed.pathname).to.eql('/gampad/ads');
-        expect(Object.keys(parsed.query)).to.eql(['sz', 'iu', 'impl', 'gdfp_req', 'env', 'output', 'unviewed_position_start', 'url', 'description_url', 'correlator', 'cust_params']);
+        expect(Object.keys(parsed.query)).to.eql(['sz', 'iu', 'impl', 'gdfp_req', 'env', 'output', 'unviewed_position_start', 'url', 'description_url', 'correlator', 'utm_source', 'utm_campaign', 'cust_params']);
```

Comment:
See this, for example -> https://github.com/theonion/bulbs-elements/blob/master/elements/bulbs-video/components/revealed.js#L210